content/homepage/section-1.md

```diff
@@ -6,4 +6,4 @@ hidden = true
 +++
 
 Tinkerbell is a bare metal provisioning engine, built and maintained by the team at <a href="https://www.packet.com/">Packet</a>.
-Tinkerbell consists of four microservices: boots, OSIE, tink, and hegel.
+Tinkerbell consists of four microservices: Boots, Hegel, OSIE, PBnJ, and Tink.
```

Comment:
Shouldn't it be five here?

Reply:
Yes, thanks. I noticed this during one of the rebases but forgot to fix, 1s.

Reply:
Good catch. I noticed during one of the rebases but forget to go back and fix. I also went looking for other instances of "[fF]our" and found other places to fix.